packages/angular_devkit/schematics/src/utility/update-buffer.ts

```diff
@@ -26,6 +29,7 @@ export class ContentCannotBeRemovedException extends BaseException {
  * it means the content itself was deleted.
  *
  * @see UpdateBuffer
+ * @deprecated Since v12.2
```

Comment:
Nit: deprecated comments should be updated to `@deprecated Since v13.0`

Reply:
Thanks for the catch. Fixed accordingly.